test/.eslintrc.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+    "env": {
+        "es6": true,
+		"mocha": true,
+		"amd": true
+    },
+    "rules": {
+        "indent": [
+            "error",
+            4
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        "quotes": [
+            "off"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ],
+		"brace-style": [
```

Comment:
There's a separate one because the environment needs to be "mocha." But I just realized that it will duplicate the rest of the rules. I'll delete the rules out of it.